src/rust/bitbox02/src/keystore.rs

```diff
@@ -558,27 +554,16 @@ mod tests {
     fn test_secp256k1_get_private_key() {
         lock();
         let keypath = &[84 + HARDENED, 0 + HARDENED, 0 + HARDENED, 0, 0];
-        assert!(secp256k1_get_private_key(keypath, false).is_err());
+        assert!(secp256k1_get_private_key(keypath).is_err());
 
         mock_unlocked_using_mnemonic(
             "abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon about",
             "",
         );
 
         assert_eq!(
-            hex::encode(secp256k1_get_private_key(keypath, false).unwrap()),
+            hex::encode(secp256k1_get_private_key(keypath).unwrap()),
             "4604b4b710fe91f584fff084e1a9159fe4f8408fff380596a604948474ce4fa3"
         );
-
-        // See first test vector in
```

Comment:
Why is this test not needed anymore?

Reply:
It tests the output of the function with the `tweak_bip86` param set to true, but this PR removes this param altogether and no bip86 tweaking is taking place here anymore.